web/src/app/monitors/monitor-create-edit/monitor-create-edit.component.ts

```diff
@@ -96,7 +96,10 @@ export class MonitorCreateEditComponent implements OnInit, OnDestroy {
   saveMonitor(uid: string, monitors: MonitorModel[]): void {
     this.saveMonitorSubscription = this.monitorService.saveMonitors(uid, monitors)
       .subscribe(
-        () => this.router.navigate([`/projects/${uid}/monitors`]),
+        () => {
+          this.monitorService.pingMonitor(this.projectUid, this.monitorUid, 'automatic');
```

Comment:
this should probably go in a `pipe` rather than the `subscribe`

Reply:
fixed it , added a check for save and update monitor scenario too.